Modules/Sources/WooFoundation/Utilities/IndefiniteCircularProgressViewStyle.swift

```diff
@@ -78,20 +100,20 @@ public struct IndefiniteCircularProgressViewStyle: ProgressViewStyle {
     }
 }
 
-private extension IndefiniteCircularProgressViewStyle {
-    enum Constants {
-        static let lineWidth: CGFloat = 10.0
-        static let backgroundOpacity: CGFloat = 0.2
+public extension IndefiniteCircularProgressViewStyle {
+    public enum Constants {
```

Comment:
nit: getting `'public' modifier is redundant for enum declared in a public extension` warning

```suggestion
    enum Constants {
```